benchmarks/mx_formats/cast_bench.py

```diff
@@ -194,6 +208,42 @@ def run(
         bytes_w = (y_d1.numel() + s_d1.numel()) * bytes_per_el_fp8
         bps = (bytes_r + bytes_w) / (time_us / 1e6)
 
+    elif mode == "dim1_mx_cuda_floor":
+        bench_fn = partial(
```

Comment:
nit: either refactor the other branches to use partial, or remove partial here, to keep the file code style consistent